### PR TITLE
Bump version of Guava and Jackson Core.

### PR DESCRIPTION
**Pull request by mattwhisenhunt**
@garrettjonesgoogle PTAL

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -595,13 +600,13 @@
     <project.jsr305.version>1.3.9</project.jsr305.version>
     <project.gson.version>2.1</project.gson.version>
     <project.jackson-core-asl.version>1.9.11</project.jackson-core-asl.version>
-    <project.jackson-core2.version>2.1.3</project.jackson-core2.version>
+    <project.jackson-core2.version>2.9.2</project.jackson-core2.version>
     <project.protobuf-java.version>2.6.1</project.protobuf-java.version>
     <project.guava.version>17.0</project.guava.version>
```
Comment (garrettjonesgoogle) [inline pom.xml]: Bump this to 19.0 like you did for oauth2

**Comment by garrettjonesgoogle** [comment]
LGTM

**Comment by Kitanotori** [comment]
Could you guys please release this? Spent quite many hours solving issues caused by guava-jdk5 dependency still remaining in 1.23.0 release.

**Comment by philix** [comment]
@garrettjonesgoogle can this be published on maven?

**Comment by mattwhisenhunt** [comment]
There is a release in the works. I'm sorry I don't know specifically when but the time scale is days not months.

**Comment by kentongray** [comment]
Thanks @mattwhisenhunt! Any update is a good update from my perspective!

**Comment by bbhoss** [comment]
Is it possible to resolve this with an exclusion until this is released? Really looking forward to it! :)

**Comment by jadekler** [comment]
This change has been released as part of 1.24.1.

**Comment by philix** [comment]
Thanks. Updated and removed exclusion rule.

**Pull request merged**